app/controllers/application_controller.rb

```diff
@@ -74,12 +74,12 @@ def routing_error
   end
 
   def not_authorized
-    render json: {error: "Error: not authorized"}, status: :unauthorized
+    render json: { error: "Error: not authorized" }, status: :unauthorized
```

Comment:
This was an automatic change, Rubocop wants spaces inside the curl braces.

Reply:
this (and the second spacing issue in this file) are my fault (a tooling issue on my end) - I disable the git hooks to commit because I'm straddling two node versions  - when I try to run husky but have node 16 loaded in the path commits fail.

Thanks for cleaning up my mess.